### ISSUE TITLE
Unwanted escaped quotes

### ISSUE THREAD

**Issue by jamesj2** [Benjythebee/esm-mailwind#1]
Esm-mailwind is converting quotes in attributes. 

```
<img src="{{ url("/images/portrait_footer.png") }}" class="object-cover mt-3">
```

After running esm-mailwind:
```
<img src="{{ url(" images="" portrait_footer.png&#x22;)="" }}&#x22;="" class="object-cover mt-3" style="margin-top: 12px; -o-object-fit: cover; object-fit: cover;">
```

**Comment by Benjythebee** [comment]
Sorry for taking so long to see this.

This is because the library doesn't quite support variables with the format `{{ }}` the variables should be processed before you call mailwind.


So I published a new version `1.0.3` which should support handlebars.

so say I have `{{ myVariable ]}` will be untouched and the output will still contain `{{ myVariable }}`.

Is this what you wanted?

**Comment by jamesj2** [comment]
Yep, that's what I was looking for.  Now I can pass it onto the template engine.  Thanks!
